### PR TITLE
Fix getTripOptions and getTrackingOptions

### PR DESCRIPTION
**Pull request by brettguenther**
Bad typings and returns not translating JSON to JS.

Fixes FENCE-1053

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Migration guides
 
+## 3.7.1 to 3.7.2
+
+- `Radar.getTripOptions()` and `Radar.getTrackingOptions` return typed `RadarTripOptions` and `RadarTrackingOptions` respectively rather than nesting within an `options` key.
```
Comment (tjulien) [inline MIGRATION.md:5]: @lmeier Can you add a migration code snippet (example below)

File: android/src/main/java/io/radar/capacitor/RadarPlugin.java [hunk 2]
```diff
@@ -476,9 +476,7 @@ public void isTracking(PluginCall call) {
     @PluginMethod()
     public void getTrackingOptions(PluginCall call) {
         RadarTrackingOptions options = Radar.getTrackingOptions();
-        JSObject ret = new JSObject();
-        ret.put("options", options != null ? options.toJson() : null);
-        call.resolve(ret);
+        call.resolve(RadarPlugin.jsObjectForJSONObject(options.toJson()));
```
Comment (tjulien) [inline android/src/main/java/io/radar/capacitor/RadarPlugin.java:479]: @lmeier Can you double check that this new return style is:
1) Consistent with the rest of the capacitor sdk (do we need fix this anywhere else?)
2) Consistent with how react native and flutter implement `getTrackingOptions()` and `getTripActions()`

Comment (tjulien) [reply android/src/main/java/io/radar/capacitor/RadarPlugin.java:479]: @lmeier Do we need to fix https://github.com/radarlabs/capacitor-radar/blob/master/src/definitions.ts ?

Or this bug fix now makes those definitions correct?

File: MIGRATION.md (hunk 1 above)
Comment (lmeier) [reply MIGRATION.md:5]: https://github.com/radarlabs/capacitor-radar/pull/86/commits/ff5525e6e1a21c8cfbd28e39d6b35e45ad6a830c

File: android/src/main/java/io/radar/capacitor/RadarPlugin.java (hunk 2 above)
Comment (lmeier) [reply android/src/main/java/io/radar/capacitor/RadarPlugin.java:479]: Verified that the rest of the SDK is consistent, and that type definitions are now correct

Comment (lmeier) [reply android/src/main/java/io/radar/capacitor/RadarPlugin.java:479]: Confirmed that react native is the same (correct).

Comment (lmeier) [reply android/src/main/java/io/radar/capacitor/RadarPlugin.java:479]: Flutter is also correct.

**Review by lmeier — APPROVED** [review]
@tjulien this lgtm now.

**Review by tjulien — APPROVED** [review]

**Pull request merged**